### ISSUE TITLE
Invalid CSS Attribute : "box-radius"

### ISSUE THREAD

**Issue by Yudaidhun** [cakephp/app#594]
This is a (multiple allowed):

* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Application Skeleton Version: 3.5.15
* Platform and Target: Firefox / Chrome.

### What you did
$this->Flash->set('foo');

### What happened
The appeared Flash has invalid attribute called "box-radius".
This is included in `/webroot/css/cake.css` .
```
div.message {
    text-align: center;
    cursor: pointer;
    display: block;
    font-weight: normal;
    padding: 0 1.5rem 0 1.5rem;
    transition: height 300ms ease-out 0s;
    background-color: #a0d3e8;
    color: #626262;
    top: 15px;
    right: 15px;
    z-index: 999;
    overflow: hidden;
    height: 50px;
    line-height: 2.5em;
    box-radius: 5px; /* this!!! */
}
```

### What you expected to happen
Well, nothing.
I think we just need to wipe that line unless it intends to set "border-radius".
I'm not really experienced with web, sorry if I'm reporting something wrong.

**Comment by markstory** [comment]
That line should be deleted. If you edit the file in github, you will have the option to open a pull request and we can get that fixed :)

**Comment by Yudaidhun** [comment]
thanks, pull request done!

**Comment by markstory** [comment]
Fixed in #596. Thank you!
